exporter/otlpexporter/config.go

```diff
@@ -26,6 +26,8 @@ type Config struct {
 
 	// Experimental: This configuration is at the early stage of development and may change without backward compatibility
 	// until https://github.com/open-telemetry/opentelemetry-collector/issues/8122 is resolved
+	//
+	// Deprecated: [v0.123.0] batch configuration moving to queue configuration.
```

Comment:
With `queue configuration`, do you mean the `sending_queue` field from this configuration?

If so, how the current values from `batcher` are configured? `flush_timeout` and the others. Because checking the current state of the code, it is not clear to me.

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/12746